scenes/tileset.tres

```diff
@@ -1206,30 +1206,37 @@ texture_region_size = Vector2i(64, 64)
 0:0/0 = 0
 0:0/0/terrain_set = 0
 0:0/0/terrain = 8
+0:0/0/physics_layer_0/polygon_0/points = PackedVector2Array(-16, 32, -32, 32, -32, -32, 32, -32, 32, -16, -16, -16)
```

Comment:
These were added in commit 80bfcd0f2ba1e6b7168cf56b9584045890134929. This and the changes below add collisions to the Void_Empty layer, in the tileset's physics layer 0, which maps to "walls" at the game level.

<img width="898" height="416" alt="Image" src="https://github.com/user-attachments/assets/18fcc912-1a60-4a63-9abf-12163d74ae49" />

I don't think this change is correct. The void is not a wall: it's a non-walkable floor, and it's marked as such in tileset physics layer 2. But also, this tileset from the main game is not actually used in the quest level that uses the void tiles – you use a copy there – so I think it can be reverted without changing your quest.